runtime/oti/j9accessbarrier.h

```diff
@@ -171,13 +171,13 @@ typedef struct J9IndexableObject* mm_j9array_t;
 	(void)0)
 #define J9OBJECT__PRE_OBJECT_STORE_ADDRESS(vmThread, object, address, value) \
 	((void)0, \
-	(OMR_GC_ALLOCATION_TYPE_SEGREGATED == (J9VMTHREAD_JAVAVM((vmThread)))->gcAllocationType) ? \
+	(OMR_GC_WRITE_BARRIER_TYPE_REALTIME == (J9VMTHREAD_JAVAVM((vmThread)))->gcWriteBarrierType) ? \
```

Comment:
this should check for OMR_GC_WRITE_BARRIER_TYPE_SATB  || OMR_GC_WRITE_BARRIER_TYPE_SATB_AND_OLDCHECK, but I guess you did not introduce them yet?

Reply:
I did introduce them in OMR but I missed this spot, thank you for catching it. I will update the pull request.

Reply:
fixed